Terran-One/cw-simulate-ui
# Drive-by fix

## Issue link

N/A

## Description

When you upload a wasm file in the welcome screen and create a simulation env, the page displays a message saying there are no chains, even when there are some.

## Test steps

1. Upload a wasm file in the welcome screen
2. Create a simulation env
3. Verify it navigates you to the _Config_ page for the new chain
4. Navigate the sidebar and check it works
5. Delete the chain and check you see _No chain selected_ copy
